api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -49,6 +49,9 @@ type KubeadmConfigStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:path=kubeadmconfigs,scope=Namespaced
```

Comment:
`namespaced` is the default if I'm reading https://book.kubebuilder.io/reference/markers/crd.html correctly. I don't really know what this is doing otherwise. Why do we need it?

Reply:
It's actually not, it'll default to empty

Reply:
oh that's fun